src/EditorFeatures/Core/FindReferences/FindReferencesCommandHandler.cs

```diff
@@ -109,14 +110,14 @@ private IStreamingFindUsagesPresenter GetStreamingPresenter()
             }
         }
 
-        private async void StreamingFindReferences(
+        private async Task StreamingFindReferencesAsync(
```

Comment:
same comment as above.  it seems better here to be async-void, to indicate that this is fire-and-forget.  Namely, it helps ensure that this code knows that i needs to handle cancellation/exceptions.  With Task-Returning, someone could easily remove the try/catch thinking it isn't necessary.

Reply:
➡️ Moved this discussion to https://github.com/dotnet/roslyn/pull/28781#discussion_r208406333